content/en/includes/task-tutorial-prereqs.md

```diff
@@ -4,5 +4,5 @@ cluster, you can create one by using
 [minikube](https://minikube.sigs.k8s.io/docs/tutorials/multi_node/)
 or you can use one of these Kubernetes playgrounds:
 
-* [Katacoda](https://www.katacoda.com/courses/kubernetes/playground)
+* [Katacoda](https://www.katacoda.com/tfogo/scenarios/k8s)
```

Comment:
I'm not sure that this is the right fix.

Do we know that https://www.katacoda.com/tfogo/scenarios/k8s will remain in place, and for how long? Katacoda is shutting down.
How much trust do we have in that user's private scenarios?